8/run-silo-docker.sh

```diff
@@ -9,9 +9,9 @@ GATEWAYPORT=3001
 SILOPORT=2001
 PRIMARYPORT=2001
 DASHBOARDPORT=8081
-MEMBERSHIPTABLE="OrleansMembership"
-AWSREGION="us-west-2"
+MEMBERSHIPTABLE=OrleansMembership
+AWSREGION=http://$ADVERTISEDIP:8042
 
 docker build -t silo-host-cluster -f ./ops/SiloHost/Dockerfile ./ &&
   docker run -it -e ADVERTISEDIP=$ADVERTISEDIP  -e GATEWAYPORT=$GATEWAYPORT -e SILOPORT=$SILOPORT -e PRIMARYPORT=$PRIMARYPORT -e DASHBOARDPORT=$DASHBOARDPORT\
-   -e PRIMARYADDRESS=$PRIMARYADDRESS -e MEMBERSHIPTABLE=$MEMBERSHIPTABLE -e AWSREGION=$AWSREGION  -p $GATEWAYPORT:$GATEWAYPORT -p $SILOPORT:$SILOPORT -p $DASHBOARDPORT:$DASHBOARDPORT --rm silo-host-cluster
\ No newline at end of file
+   -e PRIMARYADDRESS=$PRIMARYADDRESS -e MEMBERSHIPTABLE=$MEMBERSHIPTABLE -e AWSREGION=$AWSREGION  -p $GATEWAYPORT:3000 -p $SILOPORT:2000 -p $DASHBOARDPORT:$DASHBOARDPORT --rm silo-host-cluster
```

Comment:
Yeah, I'd double check for multi-silo local clusters.

Reply:
That's still okay, internal ip to container, no?

Reply:
I mean the ports being fixed to 2000 and 3000.